src/BruteForceAttacker.php

```diff
@@ -70,13 +76,23 @@ public static function run($options)
         self::$callback = $options['callback'];
         self::$skipLength = $options['skipLength'];
         self::$skipCount = $options['skipCount'];
-        self::$count = 0;
+        self::$count = $options['startsCount'];
 
         // Run
-        $length = ($options['length'] >= 1) ? intval(floor($options['length'])) : 1;
+        $length = ($options['length'] >= 1) ?  intval(floor($options['length'])) : 1;
```

Comment:
Could you revert this change? 
Thanks

Reply:
Sure!, you mean the white space before `intval(...)`.

Reply:
Well, it seems you have already done it.